Makefile.am

```diff
@@ -181,3 +182,12 @@ endif
 if ENABLE_MODULE_RECOVERY
 include src/modules/recovery/Makefile.am.include
 endif
+
+
+if ENABLE_RUST_NAIVETESTS
+$(ECC_SECP):
+	git clone https://github.com/elichai/ecc-secp256k1.git -b v0.1.0 2> /dev/null | true
+	cd ecc-secp256k1 && cargo build --release --features=ffi
+	cp ./ecc-secp256k1/target/release/libecc_secp256k1.a  .
+	cp ./ecc-secp256k1/ecc_secp256k1.h  ./src/
```

Comment:
Copying libraries and header files around will be confusing. Can't you avoid it by directly linking to the files in the `ecc-secp256k1` dir?

Reply:
I thought it was simpler by yeah it's possible to link directly into the files there.
about the header, don't you want it to be under source control here?